wurst/systems/core/Tribe.wurst

```diff
@@ -125,7 +126,8 @@ public class Tribe
             TribeBoard.findForPlayer(member).board.display(member, false)
 
             // Enabling the score board, maximizing it so the player doesn't miss it
-            scoreBoard.board..display(true)..maximize(member)
+            // scoreBoard.board..display(true)..maximize(member)
```

Comment:
Why leave this commented? Just remove if it the logic was a bug.

Reply:
You're right, forgot about that.